dashboard/test/helpers/levels_helper_test.rb

```diff
@@ -267,4 +267,22 @@ def stub_country(code)
     assert_equal false, app_options[:level]['submittable']
   end
 
+  test 'Milestone posting is disabled if Gatekeeper flag is set' do
```

Comment:
I think it's a good idea to put a Gatekeeper.clear in the setup just to ensure there is no accidental state leakage across tests.